python/snewpy/models/base.py

```diff
@@ -9,6 +9,7 @@
 from scipy.special import loggamma
 
 from snewpy.neutrino import Flavor
+from snewpy.flavor_transformation import NoTransformation, FlavorTransformation
```

Comment:
```suggestion
from snewpy.flavor_transformation import NoTransformation
```
I don’t think `FlavorTransformation` is used anywhere in this PR?